src/JEM-wschema.html

```diff
@@ -1728,788 +1728,788 @@
                                                              }
                                                          }
                                                     },
-                                                    "order":["roiMain", "roiMajor", "roiMinor"],
+                                                    "order":["roiSuper", "roiMajor", "roiMinor"],
                                                     "data": [{
                                                          "value": "VISp1",
                                                          "attributes": {
-                                                             "roiMain": "CTX",
+                                                             "roiSuper": "Cortex",
```

Comment:
Just a semantic detail, but if we're going with "Subcortical" this should probably be "Cortical"?